endoreg_db/views/requirement/lookup.py

```
@@ -0,0 +1,92 @@
+
+        # PATCH
+        updates = request.data.get("updates", {})
+        if not isinstance(updates, dict):
+            return Response({"detail": "updates must be an object"}, status=status.HTTP_400_BAD_REQUEST)
+
+        store.set_many(updates)
+
+        if any(key in self.INPUT_KEYS for key in updates.keys()):
+            try:
+                recompute_lookup(pk)
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)